### PR TITLE
Create Classes model

### PR DESCRIPTION
**Pull request by kidzegeye**
## Overview

Created the Classes model for storing group classes

## Changes Made

* Created Classes model, with a many-to-many relationship with Gyms

## Next Steps (delete if not applicable)

* Create the webscraper for the classes

### CODE REVIEW

File: src/models/classes.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+class Class(Base):
+    __tablename__ = "class"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(), nullable=False)
+    description = Column(String(), nullable=False)
+    gyms = relationship(
+        "Gym",
+        secondary=classes_to_gyms,
+        back_populates="classes",  ##Change gym to rec_center
```
Comment (joycrwu) [inline src/models/classes.py:36]: prob remove the comment

**Review by joycrwu — APPROVED** [review]
other than the small comment looks gud!

File: src/models/price.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from database import Base
```
Comment (isabellahoie) [inline src/models/price.py:1]: Wait why were prices added into this PR? Do we need prices for classes?

**Review by isabellahoie — CHANGES_REQUESTED** [review]
Just the changes Joyce requested and then my question at the bottom

File: src/models/price.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+class Price(Base):
+    __tablename__ = "price"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(), nullable=False)
+    cost = Column(Integer, nullable=False)
+    one_time = Column(Boolean, nullable=False)
+    image = Column(String(), nullable=True)
+
+    def __init__(self, **kwargs):
```
Comment (isabellahoie) [inline src/models/price.py:15]: missing self.id in init

File: src/models/price.py (hunk 2 above)
Comment (kidzegeye) [reply src/models/price.py]: I removed it

**Review by isabellahoie — APPROVED** [review]
Looks good to me

**Pull request merged**